subprojects/gradle/runner/service/src/main/kotlin/com/avito/runner/service/worker/device/adb/Result.kt

```diff
@@ -0,0 +1,46 @@
+package com.avito.runner.service.worker.device.adb
+
+import org.funktionale.tries.Try
+
+sealed class Result<T> {
```

Comment:
internal?

Reply:
https://github.com/avito-tech/avito-android/pull/826#discussion_r587345541
Not much sense with this comment in context